.github/workflows/nodejs.yml

```diff
@@ -24,9 +24,14 @@ jobs:
       run: |
         npm install
 
+    - name: Setup Yarn
+      run: |
+        npm uninstall -g yarn
+        npm i -g yarn@1.22.10    
+
     - name: Lint
       run: |
-        npm run lint
```

Comment:
How come? If yarn isn't a dev dependency?

Reply:
I don't quite understand the question. To me, `npm run lint ` will run standard lint. But I want to setup some rules and for future management. So I switch to use `yarn lint`.

Reply:
I will add yarn as devDependencies.